codelabs/gRPC_Cpp_OpenTelemetry_Plugin/.bazelrc

```diff
@@ -0,0 +1,182 @@
+# bazelrc file copied from https://github.com/grpc/grpc/blob/v1.65.x/tools/bazel.rc
+
+# Bazel 7 begins to enable module by default which breaks gRPC build.
+# Therefore, this option is disabled until gRPC works with bzlmod.
+common --enable_bzlmod=false
+
+# for platform-appropriate cxxopts
+common --enable_platform_specific_config
+
+build:windows --cxxopt='/std:c++14'
+build:windows --host_cxxopt='/std:c++14'
+build:linux   --cxxopt='-std=c++14'
+build:linux   --host_cxxopt='-std=c++14'
+build:macos   --cxxopt='-std=c++14'
+build:macos   --host_cxxopt='-std=c++14'
+build:freebsd   --cxxopt='-std=c++14'
+build:freebsd   --host_cxxopt='-std=c++14'
+
+# TODO(veblush): Remove these once upb is upgraded to 25.x or later
```

Comment:
since Esun is not participating in this repo, would this make more sense? 
```suggestion
# TODO(yashkt): Remove these once upb is upgraded to 25.x or later
```